io.github.daviddesimone.opencloudsaves.yml

```diff
@@ -0,0 +1,68 @@
+app-id: io.github.daviddesimone.opencloudsaves
+runtime: org.gnome.Platform
+runtime-version: '42'
```

Comment:
this runtime will be EOL in may

43 is the current one.

Reply:
Thank you for the review - I have updated to 43.